lib/views/home/home_screen.dart

```diff
@@ -22,13 +22,6 @@ enum NavBarButton with StatelessEnum {
     screen: ManageUsers(),
   ),
 
-  /// Admins can edit the livestream schedule.
-  schedule(
-    outlined: Icon(Icons.calendar_month_outlined),
-    filled: Icon(Icons.calendar_month),
-    screen: Schedule(),
-  ),
```

Comment:
I moved this guy down since I think it looks kinda nice for "watch live" to be the first option for participants & directors.